### PR TITLE
fix: minor fixes to some informational findings

### PR DESCRIPTION
**Pull request by lekhovitsky**
Fixes:
* https://github.com/spearbit-audits/review-gearbox/issues/30
* https://github.com/spearbit-audits/review-gearbox/issues/35
* https://github.com/spearbit-audits/review-gearbox/issues/45
* https://github.com/spearbit-audits/review-gearbox/issues/67
* https://github.com/spearbit-audits/review-gearbox/issues/108

### CODE REVIEW

File: contracts/credit/CreditManagerV3.sol [hunk 1]
```diff
@@ -1304,6 +1303,11 @@ contract CreditManagerV3 is ICreditManagerV3, SanityCheckTrait, ReentrancyGuardT
         return amount;
     }
 
+    /// @dev Internal wrapper for `creditAccount.safeTransfer` call to redice contract size
```
Comment (StErMi) [inline contracts/credit/CreditManagerV3.sol]: @lekhovitsky there's a typo here: `redice` → `reduce`

**Comment by StErMi** [comment]
@lekhovitsky I think that the usage of the `USE_SAFE_PRICES_FLAG` flag (see issue https://github.com/spearbit-audits/review-gearbox/issues/108) should **also** be documented in the `CreditFacadeV3` itself and **not only** on the `PriceOracleV3`.

The `PriceOracleV3` is where the safe price behavior is configured, but it's the `CreditFacadeV3` contract that will tell the oracle to use them or not. 

Another issue about the safe price: with the PR https://github.com/Gearbox-protocol/core-v3/pull/268 now also the "adapter logic" of the `multicall` could enable the `USE_SAFE_PRICES_FLAG` flag. This scenario is not explicitly documented in the PR and should be.

TLDR: Gearbox should:
- Document the usage of the "Safe Price" mechanism also in the `CreditFacadeV3` contract
- Add to both the documentation of "Safe Price" in the `CreditFacadeV3` and `PriceOracleV3` contracts, the scenario enabling the `USE_SAFE_PRICES_FLAG` flag because of an external call to the `adapter` in `multicall`

**Comment by StErMi** [comment]
The code looks good and can be merged once the issues described in the above comments are integrated in this PR.

**Comment by lekhovitsky** [comment]
@StErMi Fixed a typo + added a comment about additional checks in `_multicall`'s NatSpec in the latest commit
